app/models/gemini_connection.rb

```diff
@@ -199,11 +200,5 @@ def oauth2_config
     def encryption_key(key: Rails.application.secrets[:attr_encrypted_key])
       [key].pack("H*")
     end
-
```

Comment:
This is an oversight from other work.  The method was moved to the `Oauth2::AuthorizationCodeBase` parent, and is unnecessary here.